network-api/app/networkapi/templates/pages/landingpage.html

```diff
@@ -53,22 +53,27 @@
         </div>
       </div>
       <div class="container">
-        <div class="row py-5 justify-content-center">{% if page.landingpage.signup %}
-          <div class="col-lg-11">
+        <div class="row">
+          <div class="py-5 {% if page.parent.title != "Opportunity" or page.children.all.count > 0 %}col-8 offset-3{% else %}col-12{% endif %}">
             <h1 class="h1-white">{{ page.landingpage.header }}</h1>
-          </div>{% else %}
-          <div class="col-12 col-md-8 mb-5">
-            <h1 class="h1-white">{{ page.landingpage.header }}</h1>
-          </div>{% endif %}
+          </div>
         </div>
-        <div class="row justify-content-around">{% if page.landingpage.signup %}
-          <div class="cms col-md-6 col-lg-5 mb-5">{{ page.landingpage.content | safe }}</div>
-          <div class="col-md-5 col-lg-5">
-            <div class="row">
-              <div class="join-us half-width col" data-cta-header="{{ page.landingpage.signup.header | escape }}" data-cta-description="{{ page.landingpage.signup.description | escape }}"></div>
+        <div class="row mb-5">{% if page.parent.title != "Opportunity" or page.children.all.count > 0 %}
```

Comment:
it seems like `branch_level` works inside the child template, so can the check for level/"should we show the side menu in the first place" happen inside that?

Reply:
I have tried a *ton* of approaches, including that one, and there are issues with it as well. This is really the best I could come up with after many hours of hammering on it.